.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,25 @@
+name: Pull request
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  tests:
+    name: Test
+    uses: swiftlang/github-workflows/.github/workflows/swift_package_test.yml@macos-support
+    with:
+      linux_swift_versions: '["nightly-main"]'
+      windows_swift_versions: '["nightly-main"]'
+      enable_macos_checks: true
+      macos_xcode_versions: '["16.2"]'
```

Comment:
Ideally we would want a `macOS 13` VM to test backwards compatibility (since this package needs to support macOS 13, which does not have `Span`) and also a `nightly` to test Span integration. But having nightly for now is enough.

Reply:
We currently don't support backwards deployment.